src/pages/my_registrations/MyRegistrations.tsx

```diff
@@ -58,7 +58,7 @@ export const MyRegistrations = ({ selectedUnpublished, selectedPublished }: MyRe
               {t('common.result_registrations')}
             </Typography>
             <MyRegistrationsList
-              registrations={filteredRegistrations.length > 0 ? filteredRegistrations : registrations}
+              registrations={filteredRegistrations.length > 0 ? filteredRegistrations : []}
```

Comment:
Hvorfor denne endringen?

Ser ut dette burde holde. om jeg ikke misser noe:
```suggestion
              registrations={filteredRegistrations}
```